app/classifier/tasks/drawing/summary.cjsx

```diff
@@ -8,6 +10,13 @@ module.exports = React.createClass
     annotation: null
     expanded: false
 
+  getCorrectSingularOrPluralOfDrawingType: (type, number) ->
+    if number>1 then "#{type}s" else type
+
+  stripMarkdownFromLabel: (label) ->
+    label = label.replace /\!\[[^\]]*\]\([^)]*\)/g, ""
+    remark.process(label)
```

Comment:
This will return empty if an answer for a question is just an image, so we may have to revisit this in future.